components/TimeoutModal.tsx

```diff
@@ -55,7 +55,12 @@ export const TimeoutModal: React.FC<TimeoutModalProps> = ({ timedOut, userArrive
     // And at the end, send back to EDD
     setTimeout(() => {
       if (typeof window !== 'undefined') {
-        const eddLoginLink = getUrl('edd-log-in')?.concat(encodeURIComponent(window.location.toString()))
+        // Note that the concatenated portion of this link is functionally useless, as IDM is not currently
+        // able to redirect based on the resource_url parameter concatenated.
+        const eddLoginLink = getUrl('bpo-log-in')?.concat(
+          '?resource_url=',
```

Comment:
I assume this addition is intentional? I see it is being added in this PR and want to double check

Reply:
Yes, it was added here and removed from the URL in urls.json